.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,51 @@
+name: PR Labeler
+on: [pull_request]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/github-script@v6
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
The github token should be something made available via the permissions field at the top level, so I don't think it should be required, but I can be wrong here.